### PR TITLE
Add landing page

### PR DESCRIPTION
**Pull request by shankariyerr**
@sindresorhus I saw your comment on Caprine and added meta tags, here is the updated site: http://touchbar.surge.sh/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+						<a class="button" id="touchBar">Download</a>
+					<pre><code>brew update && brew cask install touch-bar-simulator</code></pre>
+				</div>
+				<div class="is-6 center">
+					<img src="assets/images/screengrab.jpeg" class="screenshot" width="100%"/>
+				</div>
+			</div>
+		</div>
+		<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.0.0/jquery.min.js"></script>
+		<script src="assets/js/index.js"></script>
```
Comment (sindresorhus) [inline index.html]: You can place it in head if you use the `defer` attribute.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+		<title>Touch Bar Simulator</title>
+		<meta name="description" content="The macOS Touch Bar Simulator as a standalone app">
+		<meta name="author" content="Sindre Sorhus">
+		<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+		<meta property="og:url" content="https://github.com/sindresorhus/touch-bar-simulator">
+		<meta property="og:title" content="Touch Bar Simulator">
+		<meta property="og:description" content="The macOS Touch Bar Simulator as a standalone app">
+		<meta property="og:image" content="https://sindresorhus.com/media/touch-bar-simulator.png">
+		<link rel="shortcut icon" href="favicon.png">
+		<link rel='stylesheet prefetch' href='https://unpkg.com/griddd@1.0.2/dist/griddd.min.css'>
```
Comment (sindresorhus) [inline index.html]: Double-quotes

File: index.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+	<head>
+		<meta charset="UTF-8">
+		<title>Touch Bar Simulator</title>
+		<meta name="description" content="The macOS Touch Bar Simulator as a standalone app">
+		<meta name="author" content="Sindre Sorhus">
+		<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+		<meta property="og:url" content="https://github.com/sindresorhus/touch-bar-simulator">
+		<meta property="og:title" content="Touch Bar Simulator">
+		<meta property="og:description" content="The macOS Touch Bar Simulator as a standalone app">
+		<meta property="og:image" content="https://sindresorhus.com/media/touch-bar-simulator.png">
```
Comment (sindresorhus) [inline index.html]: Can you add Twitter ones too? Same as https://github.com/sindresorhus/caprine/blob/615ac800b3fe3be3d8521a78743e07a179c72b05/index.html#L10-L14

File: index.html [hunk 4]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+	<head>
+		<meta charset="UTF-8">
+		<title>Touch Bar Simulator</title>
+		<meta name="description" content="The macOS Touch Bar Simulator as a standalone app">
+		<meta name="author" content="Sindre Sorhus">
+		<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
```
Comment (sindresorhus) [inline index.html]: `shrink-to-fit=no` is no longer needed

**Comment by shankariyerr** [comment]
@sindresorhus ah, silly me! Fixed.

**Comment by sindresorhus** [comment]
Woot. Thank you @shankariyerr 🙌

**Comment by shankariyerr** [comment]
Awesome! This is my first contribution to any of your repos, thanks for letting me do this. Looking forward to more of this.

**Pull request merged**